### PR TITLE
Rename "you're caught up" carousel references

### LINKED ISSUE #643: "You're caught up" is no longer an accurate descriptor
### Platform

MacOS 10.14.6 x86

### Browser

Edge

### Addon version

v0.20.1

### Details

We still refer to the "You're caught up" element as such in the description of its hiding checkbox in Tweaks, but it no longer has that text, so new users won't know what the option refers to!

What should it be called, that would be clear and obvious? I have absolutely no idea. It's, you know, super great.

### PR DESCRIPTION
**Pull request by marcustyphoon**
I don't know. I hate this. Can we put a picture of the element in question inline in the settings panel?

#### Issues this closes
resolves #643

### CODE REVIEW

**Comment by AprilSylph** [comment]
Tumblr still doesn't have a good name for this internally.

How about briefly describing it? "The Changes/Shop/etc. links carousel" is unambiguous.

**Comment by marcustyphoon** [comment]
Hmm. They all seem... fine.

| |
| - |
| Screen Shot 2022-08-27 at 3 08 30 PM | 
| Screen Shot 2022-08-27 at 3 07 58 PM | 
| Screen Shot 2022-08-27 at 3 07 32 PM | 
| Screen Shot 2022-08-27 at 3 06 49 PM | 
| Screen Shot 2022-08-27 at 3 06 02 PM |

File: src/scripts/accesskit.json [hunk 1]
```diff
@@ -16,7 +16,7 @@
     },
     "boring_tag_chiclets": {
       "type": "checkbox",
-      "label": "De-animate the \"You're caught up\" carousel",
+      "label": "De-animate the dashboard \"caught up\" carousel",
```
Comment (AprilSylph) [inline src/scripts/accesskit.json:19]:
```suggestion
      "label": "De-animate the Changes/Shop/etc. links carousel",
```

File: src/scripts/tweaks.json [hunk 2]
```diff
@@ -31,7 +31,7 @@
     },
     "caught_up_line": {
       "type": "checkbox",
-      "label": "Turn \"You're caught up\" into a separating line",
+      "label": "Turn the dashboard \"caught up\" carousel into a separating line",
```
Comment (AprilSylph) [inline src/scripts/tweaks.json:34]:
```suggestion
      "label": "Turn the Changes/Shop/etc. links carousel into a separating line",
```

**Review by AprilSylph — CHANGES_REQUESTED** [review]

**Review by AprilSylph — APPROVED** [review]

**Pull request merged**